generate-types/src/main.rs

```diff
@@ -0,0 +1,26 @@
+use schemars::schema_for;
+use std::fs;
+
+fn main() {
+    let schema_config_device_key_proxy = schema_for!(device_key_proxy::UserConfig);
+    fs::write(
+        "./device_key_proxy_config_type.txt",
+        format!(
+            "{:?}",
+            serde_json::to_vec(&schema_config_device_key_proxy)
+                .expect("error converting user config for device key proxy")
+        ),
+    )
+    .expect("Failed to write to device key proxy config");
```

Comment:
To my comment above, I would do something like this instead

```suggestion
    println!(
        "{}",
        serde_json::to_string_pretty(&schema_config_device_key_proxy)
            .expect("error converting user config for device key proxy")
    );
```